network/config/core.yaml

```diff
@@ -498,7 +498,7 @@ vm:
         # https://docs.docker.com/engine/admin/logging/overview/
         # Note: Set LogConfig using Environment Variables is not supported.
         hostConfig:
-            NetworkMode: host
+            NetworkMode: docker_test
```

Comment:
I'm not sure if it's a good idea to change the network name. Let's discuss this during the next call?